### ISSUE TITLE
[Question] How do you manually commit the offset after processing each record

### ISSUE THREAD

**Issue by atiqsayyed** [monix/monix-kafka#24]
Hi,
I'm trying to achieve manual commit for each record, these are my settings
```
  val consumerCfg = KafkaConsumerConfig.default.copy(
    bootstrapServers = List("localhost:9092"),
    groupId = "monixGroup",
    enableAutoCommit = false,
    observableCommitType = ObservableCommitType.Sync,
    observableCommitOrder = ObservableCommitOrder.AfterAck
  )
```
Following is my consumer Observable
```
 val monixConsumer = KafkaConsumerObservable[String, String](consumerCfg, List("monix"))
    val consumer = monixConsumer
    .bufferTimedWithPressure(1.second, 5)
    .map(deserialize(_))
    .map(callApi(_))
    .runAsyncGetLast

```
It only commits the offset once it's done processing all the elements, not the batch of 5.
I want to commit every batch manually, is there a way todo this?

**Comment by Avasil** [comment]
Hi,
Unfortunately there is no ```Observable``` like this (```monix-kafka``` is very light wrapper) but you could disable commits and do it manually with ```consumer.commitSync()```

You can create your own consumer using ```KafkaConsumerObservable.createConsumer()``` and there is ```KafkaConsumerObservable.apply``` accepting it in the constructor.

The easiest way to disable commits would be if @alexandru releases new version containing merged PR #14, then you could use ```monix.observable.commit.order = "no-ack"```

**Comment by Avasil** [comment]
@poslegm 
Addition like this would be greatly appreciated! If you create PR we can work out the implementation details. :)
